schema.json

```diff
@@ -0,0 +1,131 @@
+{
+  "title": "JSON schema for Release Drafter yaml files",
+  "$id": "https://github.com/release-drafter/release-drafter/blob/master/schema.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
I'm by no means an expert in JSON Schema, but I noticed the latest version is draft-07. Do you happen to know why [`joi-to-json-schema`](https://github.com/lightsofapollo/joi-to-json-schema) doesn't support draft-07?
Also if we're to conform to draft-04 I think we have to change `$id` to `id` as that is what changed in draft-06: https://json-schema.org/draft-06/json-schema-release-notes.html.

Reply:
`joi-to-json-schema` currently only supports draft-04 as stated in the readme.

We could raise a issue asking for draft-07 support